src/logplex_http_drain.erl

```diff
@@ -113,6 +115,12 @@ valid_uri(#ex_uri{scheme=Http,
 valid_uri(_) ->
     {error, invalid_http_uri}.
 
+is_compressed(#ex_uri{fragment=Fragment}) when is_list(Fragment) ->
+    lists:member("gzip", string:tokens(Fragment, "+"));
```

Comment:
At the moment `gzip` will be the only supported flag on https drains. I say we punt on deciding upon the separator token for now (my vote would be for `&` characters).

```erlang
is_compressed(#ex_uri{fragment="gzip"}) ->
```

This is the same approach that was taken for TLS Syslog drains and the `insecure` option.